src/OrchardCore/OrchardCore.Abstractions/Localization/LocalizationCookieName.cs

```diff
@@ -0,0 +1,19 @@
+using Microsoft.AspNetCore.Localization;
+
+namespace OrchardCore.Localization;
+
+/// <summary>
+/// Contains a localization cookie names.
+/// </summary>
+public class LocalizationCookieName
```

Comment:
Maybe `LocalizationCookieNames`

Reply:
I expect someone will suggest that ;) I remembered the guideline from Microsoft long time back to use singular instead such as `EnvironmentName`

Reply:
Okay then